chaincode-labs/chaincode-podcast/block-building.md

```diff
@@ -1,47 +1,54 @@
 ---
-title: "Block Building"
-transcript_by: kouloumos via tstbtc v1.0.0 --needs-review
-media: https://podcasters.spotify.com/pod/show/chaincode/episodes/Block-Building-with-Clara-and-Murch---Episode-18-e1dmitu
-tags: ['research', 'mining','bitcoin-core']
-speakers: ['Clara Shikhelman']
-categories: ['podcast']
-date: 2022-02-01
+title: 'Block Building'
+transcript_by: 'markon1-a via review.btctranscripts.com'
+media: 'https://podcasters.spotify.com/pod/show/chaincode/episodes/Block-Building-with-Clara-and-Murch---Episode-18-e1dmitu'
+date: '2022-02-01'
+tags:
+  - 'research'
+  - 'mining'
+  - 'bitcoin-core'
+speakers:
+  - 'Clara Shikhelman'
+  - 'Patrick Murck'
+categories:
+  - 'podcast'
 episode: 18
-summary: "Postdoc Researcher Clara joins Murch to discuss their block building research. They cover their proposal, which outlines suggested improvements to the current Bitcoin Core block building algorithm using candidate sets"
+summary: 'Postdoc Researcher Clara joins Murch to discuss their block building research. They cover their proposal, which outlines suggested improvements to the current Bitcoin Core block building algorithm using candidate sets'
 ---
-Speaker 0: 00:00:00
+Patrick Murck: 00:00:00
 
 The idea is we want the set of miners to be an open set where anybody can enter and exit as they wish.
-And if we now had obvious optimizations that people had to implement to be as competitive as possible, That would make it harder for new miners to enter this place.
-Hey, Murch.
+And if we now had obvious optimizations that people had to implement to be as competitive as possible, that would make it harder for new miners to enter this place.
+
 
 Speaker 1: 00:00:32
```

Comment:
I observed that the host's name has not been attributed. Could you please address this omission?
Please take time to familiarize yourself with hosts to help you in future reviews.
The name of the host for this current podcast is "Adam Jonas"